example/presto_standalone/main.tf

```diff
@@ -0,0 +1,151 @@
+locals {
+  nomad_namespace   = "default"
+  nomad_datacenters = ["dc1"]
+
+  presto = {
+    shared_secret_provider   = "user"
```

Comment:
Shouldn't this be equal to "vault"?

Reply:
The current module version does not support vault provided credentials https://github.com/fredrikhgrelland/terraform-nomad-presto/issues/64

Reply:
Correct!